{"name": "test_fixture_zcc[cc_calculator_shuju3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0.30000000000000004 == 0.3\n  +0.30000000000000004\n  -0.3", "trace": "self = <cc_work.test_fixture_zcc.TestCal_cc_new object at 0x00000000039B4D30>, cc_calculator_func = <cc_work.calculator_cc.Calculator object at 0x00000000039B4358>, cc_calculator_shuju = [0.1, 0.2, 0.3]\n\n    @pytest.mark.run(order=1)\n    def test_fixture_zcc(self, cc_calculator_func, cc_calculator_shuju):\n        try:\n            result = cc_calculator_func.add(cc_calculator_shuju[0], cc_calculator_shuju[1])\n            print('这是测试用例1')\n            print(f'实际值为{result}')\n            print(f'预期值为{cc_calculator_shuju[2]}')\n            # assert result == cc_calculator_shuju[2]  写在else里才能断言成功\n        except Exception as e:\n            print(e)\n        else:\n>           assert result == cc_calculator_shuju[2]\nE           assert 0.30000000000000004 == 0.3\nE             +0.30000000000000004\nE             -0.3\n\ntest_fixture_zcc.py:25: AssertionError"}, "parameters": [{"name": "cc_calculator_shuju", "value": "[0.1, 0.2, 0.3]"}], "start": 1609398357893, "stop": 1609398357894, "uuid": "1378e722-fc27-48dd-8160-3f724e12de95", "historyId": "d18a65c205c0ea60cc51d066823ca1c5", "testCaseId": "53c508109c33f2121310f004fb063ffe", "fullName": "test_fixture_zcc.TestCal_cc_new#test_fixture_zcc", "labels": [{"name": "suite", "value": "test_fixture_zcc"}, {"name": "subSuite", "value": "TestCal_cc_new"}, {"name": "host", "value": "admin-PC"}, {"name": "thread", "value": "12680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture_zcc"}]}